### ISSUE TITLE
add a plain javascript example(ie, no frameworks)

### ISSUE THREAD

**Issue by komuw** [bufbuild/connect-web-integration#10]
Is it possible to use `connect-web` with just javascript without any framework? If yes, I would appreciate an example been added to this repo.

Thanks.

**Comment by timostamm** [comment]
That is certainly possible. We have such an example in the connect-web repository [here](https://github.com/bufbuild/connect-web/tree/main/packages/example). Is this what you are looking for, or were you thinking of something different?

We will look into making this example available here as well. Thanks for the feedback!

**Comment by timostamm** [comment]
We are tracking this in https://linear.app/bufbuild/issue/TCN-334/add-a-vanilla-typescript-example-to-connect-web-integration

**Comment by komuw** [comment]
> That is certainly possible. We have such an example in the connect-web repository [here](https://github.com/bufbuild/connect-web/tree/main/packages/example). Is this what you are looking for, or were you thinking of something different?
> 
> We will look into making this example available here as well. Thanks for the feedback!

Thanks. That would be a good start.
Q: that example generates typescript files, is it possible to also generate js instead?

> We are tracking this in ...

I don't have access to that link, but I'm subscribed to this github issue and will get notified when it is closed/resolved

**Comment by timostamm** [comment]
> Q: that example generates typescript files, is it possible to also generate js instead?

Yes, you can simply remove the `target` plugin option to do so. See [here](https://github.com/bufbuild/connect-web/tree/main/packages/protoc-gen-connect-web#plugin-options) for an explanation of this option.

**Comment by timostamm** [comment]
> that example generates typescript files, is it possible to generate js instead?

Yes, you can simply set the plugin option `target=js`. See [here](https://github.com/bufbuild/connect-web/tree/main/packages/protoc-gen-connect-web#plugin-options) for an explanation of this option. 

TypeScript is a superset of ECMAScript, it simply adds a type system on top of JavaScript. Everything you can do in TypeScript, you can also do in plain JavaScript. 

I encourage you to give the example a try! Just change the plugin option to generate JavaScript, and rename `index.ts` to `index.js`. You will have to remove a few pieces of TypeScript from `index.js` so that it becomes valid JavaScript, but I can guarantee you that it works :)
